62_CAD/vertex_shader.hlsl

```diff
@@ -455,6 +455,30 @@ PSInput main(uint vertexID : SV_VertexID)
         //outV.setMinCurvePrecomputedRootFinders(PrecomputedRootFinder<float>::construct(curveMinRootFinding));
         //outV.setMaxCurvePrecomputedRootFinders(PrecomputedRootFinder<float>::construct(curveMaxRootFinding));
     }
+    else if (objType == ObjectType::FONT_GLYPH)
+    {
+        float64_t2 topLeft = vk::RawBufferLoad<double2>(drawObj.geometryAddress, 8u);
+        float64_t2 dirU = vk::RawBufferLoad<double2>(drawObj.geometryAddress + sizeof(double2), 8u);
+        float64_t2 dirV = vk::RawBufferLoad<double2>(drawObj.geometryAddress + sizeof(double2) * 2, 8u);
+        uint32_t textureId = vk::RawBufferLoad<uint32_t>(drawObj.geometryAddress + sizeof(double2) * 3, 4u);
+
+        const float2 screenTopLeft = (float2) transformPointNdc(clipProjectionData.projectionToNDC, topLeft);
+        const float2 screenDirU = (float2) transformVectorNdc(clipProjectionData.projectionToNDC, dirU);
+        const float2 screenDirV = (float2) transformVectorNdc(clipProjectionData.projectionToNDC, dirV);
+
+        float2 corner = float2(bool2(vertexIdx & 0x1u, vertexIdx >> 1));
+        const float2 coord = screenTopLeft + corner * (screenDirU + screenDirV);
```

Comment:
This should be topLeft + corner.x * dirU + corner.y * dirV
Now rotation is handled

Reply:
Done